src/applications/accredited-representative-portal/components/common/Header/MobileHeader/MobileOfficialGovtWebsite.jsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/button-has-type */
+import React, { useState } from 'react';
+import recordEvent from '~/platform/monitoring/record-event';
+
+export const MobileOfficialGovtWebsite = () => {
+  const [expanded, setExpanded] = useState(false);
+
+  const onToggle = () => {
+    if (expanded) {
+      recordEvent({ event: 'int-accordion-collapse' });
+    } else {
+      recordEvent({ event: 'int-accordion-expand' });
+    }
+
+    setExpanded(!expanded);
+  };
+
+  return (
+    <div className="vads-u-display--flex vads-u-flex-direction--column">
+      {/* Banner */}
+      <div className="vads-u-background-color--gray-lightest vads-u-display--flex vads-u-align-items--center vads-u-justify-content--center vads-u-text-align--center vads-u-padding--0p5">
+        <img
+          alt="U.S. flag"
+          className="header-us-flag vads-u-margin-right--1"
+          src="https://www.va.gov/img/tiny-usa-flag.png"
+        />
+        {/* eslint-disable-next-line @department-of-veterans-affairs/prefer-button-component, react/button-has-type */}
+        <button
```

Comment:
I think you can add `type="button"` here to get around disabling eslint for the `react/button-has-type` warning... the `@department-of-veterans-affairs/prefer-button-component` warning will have to remain, though :/